rmgpy/rmg/main.py

```diff
@@ -275,6 +275,9 @@ def load_input(self, path=None):
         if self.quantum_mechanics:
             self.reaction_model.quantum_mechanics = self.quantum_mechanics
 
+        for reaction_system in self.reaction_systems:
```

Comment:
I don't really understand why we need this and how it fixes the issue described in the commit message?

Reply:
in `remove_species_from_edge` function in rmg.model:
```
        # clean up species references in reaction_systems
        for reaction_system in reaction_systems:
            try:
                reaction_system.species_index.pop(spec)
            except KeyError:
                pass

            # identify any reactions it's involved in
            rxn_list = []
            for rxn in reaction_system.reaction_index:
                if spec in rxn.reactants or spec in rxn.products:
                    rxn_list.append(rxn)

            for rxn in rxn_list:
                reaction_system.reaction_index.pop(rxn)
```
When species are removed from the edge, we remove it from the reaction systems attached to the `CoreEdgeReactionModel`.  The reaction systems are added to the main RMG object in rmg.input (rmg.reaction_systems), but they also need to be added to `rmg.reaction_model.reaction_systems` (which this commit does) so we can remove the edge species.  Otherwise, `rmg.reaction_model.reaction_systems` is an empty list, so we don't actually remove the reaction systems.

Reply:
Okay that makes sense. Can you change the commit message to reflect that? Other than that I think this looks good to me.